src/pytest_postgresql/executor.py

```diff
@@ -136,19 +136,24 @@ def init_directory(self):
         # remove old one if exists first.
         self.clean_directory()
         init_directory = (
-            self.executable, 'initdb',
-            '-o "--auth=trust --username=%s"' % self.user,
-            '-D %s' % self.datadir,
+            self.executable, 'initdb', '-D {}'.format(self.datadir)
         )
 
         if self.password:
-            with tempfile.NamedTemporaryFile() as password_file:
+            with tempfile.NamedTemporaryFile(mode='w') as password_file:
```

Comment:
when flushing, the mode=w is not needed.

Reply:
Yea, I've added that to make `write` function accept password as `str`. Now i saw that in your pull request you've added password encoding in init so I decided to leave the default mode. If you think it is not a good idea to leave it not working without your pr then tell me about it and I will change.